README.md

```diff
@@ -1,14 +1,49 @@
-API Example Microservice
+individuals-reliefs-api
 ========================
 
-TODO Describe microservice / API
- 
-# Run tests
+The Individuals Reliefs API allows a developer to retrieve, create and amend, and delete:
+- Relief Investments
+- Other Reliefs
+- Foreign Reliefs
+- Pension Reliefs
+
+## Requirements
+- Scala 2.12.x
+- Java 8
+- sbt > 1.3.7
+- [Service Manager](https://github.com/hmrc/service-manager)
+
+## Running the microservice
+Run from the console using: `sbt run` (starts on port 7796 by default)
+
+Start the service manager profile: `sm --start MTDFB_INDIVIDUALS_RELIEFS
+
+## Run tests
+Run unit tests: `sbt test`
+
+Run integration tests: `sbt it:test`
+
+## To view the RAML
+To view documentation locally ensure the Obligations API is running, and run api-documentation-frontend:
+
 ```
-sbt test
-sbt it:test
+./run_local_with_dependencies.sh
 ```
 
-# License
+Then go to http://localhost:9680/api-documentation/docs/api/preview and use this port and version:
+
+```
+http://localhost:7793/api/conf/1.0/application.raml
```

Comment:
does not match the port in the running the microservice section, one needs updating